Assets/Scripts/DynamicCamera/CameraController.cs

```diff
@@ -0,0 +1,60 @@
+﻿namespace HarmonyQuest.DynamicCamera
+{
+    using UnityEngine;
+    using System.Collections.Generic;
+
+    public class CameraController : MonoBehaviour
+    {
+        [SerializeField]
+        private List<CameraBehavior> behaviors = new List<CameraBehavior>();
+
+        private Vector3 target;
+
+        private static CameraController inst;
+        public static CameraController instance
+        {
+            get
+            {
+                if (!inst)
+                {
+                    inst = GameObject.FindObjectOfType<CameraController>();
```

Comment:
Should we also check for instantiating inst if it isn't already set up in Awake? Seems like we'd want to load it in as soon as possible rather than waiting for something to request it.

Reply:
Yeah, added the singleton check into `Awake()`.